src/vs/platform/terminal/test/node/terminalEnvironment.test.ts

```diff
@@ -132,7 +132,7 @@ suite('platform - terminalEnvironment', () => {
 						/.+\/out\/vs\/workbench\/contrib\/terminal\/common\/scripts\/shellIntegration-login.zsh/
 					];
 					function assertIsEnabled(result: IShellIntegrationConfigInjection, globalZdotdir = homedir()) {
-						strictEqual(Object.keys(result.envMixin!).length, 3);
+						strictEqual(Object.keys(result.envMixin!).length, 4);
```

Comment:
We can remove this length if that's the problem

Reply:
I think you are right!